bylaws/009-funding.md

```diff
@@ -0,0 +1,47 @@
+# Funding
+
+## Scope and objectives
+
+PHP-FIG is, as it often happens in Open Source communities, an organization composed by volunteers only; this doesn't exclude the fact that the organization itself may have some costs to sustain itself, like for maintaining a couple of tools such as a site and an email account. For such and similar reasons, we need a way to raise and govern money inside our organization.
+
+This document defines the roles, rules and processes that must be followed inside the PHP-FIG in all matters regarding money, to guarantee fairness, transparency and impartiality. This document intentionally describes the only allowed processes and ways to raise, administer and spend money, so that any change to such processes or any variation in budget allocation requires a bylaw change vote. 
+
+## Fiscal hosting
+For handling money and fiscal matters, the PHP-FIG will use [OpenCollective](https://opencollective.com/) as a fiscal host.
+As for any other tool pertaining to the PHP-FIG, secretary will be the only ones to have full administrative access to PHP-FIG's OpenCollective account, but they will not have any decision power over it, and they are instead bounded by this document.
+
+All monetary contributions will flow through it, so that any tax matter will be handled automatically, and every financial decision will be published and publicly available through their budgetary tools.
+
+## Raising money
+### Through OpenCollective
+Contributions will be accepted through OpenCollective; PHP-FIG members will refrain from suggesting or requiring contributions from other members or the community at large if the monetary needs of the PHP-FIG are already met (see "Spending money" section).
+
+### Through TideLift
+As the main way of raising money, PHP-FIG will use [TideLift](https://tidelift.com/).
+The TideLift account will pay out any dividend to the OpenCollective account.
```

Comment:
```suggestion
## Raising money

PHP-FIG accepts donations through multiple channels.  Only the Secretaries may actively solicit donations from any party.  No other PHP-FIG member may do so.

### Through OpenCollective

Direct contributions will be accepted through OpenCollective;.

### Through TideLift

Contributions will be accepted through [TideLift](https://tidelift.com/).  They will pay out to the OpenCollective account.

### Through Thanks.dev

Contributions will be accepted through [Thanks.dev](https://thanks.dev).  They will pay out to the OpenCollective account.
```

This puts the fundraising responsibility (such as it is) exclusively on the Secretaries, but gives us easy "pluggable" flexibility to add/remove supported channels here over time.